src/app.js

```diff
@@ -18,6 +19,10 @@ app.use((req, res, next) => {
   res.send({ error: '404 Sorry the page has not yet been defined try /api/v1/' });
 });
 
-app.listen(process.env.PORT || 3000, () => {});
+app.listen(process.env.PORT || 3000, () => {
+  migration().then(() => {
```

Comment:
Are you running your migrations any time a user connects to your app? That doesn't seem very efficient. Please explain your thought-process for this.